### PR TITLE
fix: remove the inconsistent UA string format

### PR DESCRIPTION
**Pull request by T34-active**
### Related issues and pull requests
[mdn/content#32987](https://github.com/mdn/content/pull/32987)

### CODE REVIEW

**Comment by T34-active** [comment]
@Elchi3 PTAL

File: files/en-us/web/api/workernavigator/appversion/index.md [hunk 1]
```diff
@@ -23,20 +23,20 @@ browser.
 ## Examples
 
 ```js
-alert(`Your browser version is reported as ${navigator.appVersion}`);
```
Comment (skyclouds2001) [inline files/en-us/web/api/workernavigator/appversion/index.md:26]: I think here should still be `navigator`.

@yin1999

Comment (yin1999) [reply files/en-us/web/api/workernavigator/appversion/index.md:26]: yes, the instance available in Worker should be `navigator` (aka `self.navigator`), see the [description in `WorkerNavigator` interface](https://developer.mozilla.org/en-US/docs/Web/API/WorkerNavigator).

Comment (T34-active) [reply files/en-us/web/api/workernavigator/appversion/index.md:26]: So I need to use the original code?

Comment (skyclouds2001) [reply files/en-us/web/api/workernavigator/appversion/index.md:26]: yes

File: files/en-us/web/api/workernavigator/appversion/index.md [hunk 2]
```diff
@@ -23,20 +23,20 @@ browser.
...
+alert(`Your browser version is reported as ${WorkerNavigator.appVersion}`);
```
Comment (T34-active) [inline files/en-us/web/api/workernavigator/appversion/index.md:26]:
```suggestion
alert(`Your browser version is reported as ${navigator.appVersion}`);
```

**Comment by yin1999** [comment]
You've only reverted one place, but did not do the same revert for remainings

File: files/en-us/web/api/workernavigator/appversion/index.md [hunk 3]
```diff
@@ -28,15 +28,15 @@ alert(`Your browser version is reported as ${navigator.appVersion}`);
 
 ## Notes
 
-The `navigator.userAgent` property may also contain the version
+The `WorkerNavigator.userAgent` property may also contain the version
```
Comment (T34-active) [inline files/en-us/web/api/workernavigator/appversion/index.md:31]:
```suggestion
The `navigator.userAgent` property may also contain the version
```

File: files/en-us/web/api/workernavigator/appversion/index.md [hunk 4]
```diff
@@ -28,15 +28,15 @@ alert(`Your browser version is reported as ${navigator.appVersion}`);
...
 number (for example
-"`Mozilla/5.0 (Windows; U; Win98; en-US; rv:0.9.2) Gecko/20010725 Netscape 6/6.1`"),
+"`Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0`"),
 but you should be aware of how easy it is to change the user agent string and "spoof"
 other browsers, platforms, or user agents, and also how cavalier the browser vendor
 themselves are with these properties.
 
-The `navigator.appVersion`, `navigator.appName` and
-`navigator.userAgent` properties have been used in "browser sniffing"
+The `WorkerNavigator.appVersion`, `WorkerNavigator.appName` and
```
Comment (T34-active) [inline files/en-us/web/api/workernavigator/appversion/index.md:38]:
```suggestion
The `navigator.appVersion`, `navigator.appName` and
```

File: files/en-us/web/api/workernavigator/appversion/index.md [hunk 5]
```diff
@@ -28,15 +28,15 @@ alert(`Your browser version is reported as ${navigator.appVersion}`);
...
+`WorkerNavigator.userAgent` properties have been used in "browser sniffing"
```
Comment (T34-active) [inline files/en-us/web/api/workernavigator/appversion/index.md:39]:
```suggestion
`navigator.userAgent` properties have been used in "browser sniffing"
```

File: files/en-us/web/api/workernavigator/devicememory/index.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ A floating point number; one of `0.25`, `0.5`, `1`, `2`, `4`, `8`.
 The following code can be run in a worker:
 
 ```js
-const memory = navigator.deviceMemory;
+const memory = WorkerNavigator.deviceMemory;
```
Comment (T34-active) [inline files/en-us/web/api/workernavigator/devicememory/index.md:29]:
```suggestion
const memory = navigator.deviceMemory;
```

File: files/en-us/web/api/workernavigator/globalprivacycontrol/index.md [hunk 7]
```diff
@@ -24,7 +24,7 @@ The value of the property reflects that of the {{httpheader("Sec-GPC")}} HTTP he
 ## Example
 
 ```js
-console.log(navigator.globalPrivacyControl);
+console.log(WorkerNavigator.globalPrivacyControl);
```
Comment (T34-active) [inline files/en-us/web/api/workernavigator/globalprivacycontrol/index.md:27]:
```suggestion
console.log(navigator.globalPrivacyControl);
```

File: files/en-us/web/api/workernavigator/gpu/index.md [hunk 8]
```diff
@@ -21,11 +21,11 @@ A {{domxref("GPU")}} object.
 ```js
 // Can be run inside a web worker
 async function init() {
-  if (!navigator.gpu) {
+  if (!WorkerNavigator.gpu) {
```
Comment (T34-active) [inline files/en-us/web/api/workernavigator/gpu/index.md:24]:
```suggestion
  if (!navigator.gpu) {
```

File: files/en-us/web/api/workernavigator/gpu/index.md [hunk 9]
```diff
@@ -21,11 +21,11 @@ A {{domxref("GPU")}} object.
...
     throw Error("WebGPU not supported.");
   }
 
-  const adapter = await navigator.gpu.requestAdapter();
+  const adapter = await WorkerNavigator.gpu.requestAdapter();
```
Comment (T34-active) [inline files/en-us/web/api/workernavigator/gpu/index.md:28]:
```suggestion
  const adapter = await navigator.gpu.requestAdapter();
```

File: files/en-us/web/api/workernavigator/appversion/index.md (hunk 1 above)
Comment (T34-active) [reply files/en-us/web/api/workernavigator/appversion/index.md]: so what's next??

**Pull request closed without merging**